DisguiseUnityRenderStream/Runtime/PipelineAbstraction/CameraCapture.cs

```diff
@@ -191,7 +191,7 @@ void Awake()
 
         void OnEnable()
         {
-#if (!HDRP) && (!URP)
+#if !(UNITY_PIPELINE_HDRP && HDRP) && !(UNITY_PIPELINE_URP && URP)
```

Comment:
What's the difference between UNITY_PIPELINE_HDRP and HDRP?

Reply:
`UNITY_PIPELINE_HDRP` checks that the project is using HDRP (in the project Graphics settings) and `HDRP` (now renamed to `HDRP_VERSION_SUPPORTED`) ensures minimum HDRP package version